onnx/backend/test/case/node/__init__.py

```diff
@@ -15,8 +15,74 @@
 from ..utils import import_recursive
 from ..test_case import TestCase
 
+
 _NodeTestCases = []
 
+from onnx.onnx_pb import NodeProto, AttributeProto
+from onnx.onnx_operators_pb import FunctionProto
+
+
+def function_expand_helper(node,  # type: NodeProto
+                           function_proto,  # type: FunctionProto
+                           op_prefix  # type:  Text
+                           ):  # type:  (...) -> List[NodeProto]
+    node_list = []
+    input_names_map = dict()
+    output_names_map = dict()
+    attribute_map = node.attribute
+
+    for idx in range(len(node.input)):
+        if idx not in range(len(function_proto.input)):
+            return []
+        input_names_map[function_proto.input[idx]] = node.input[idx]
+
+    for idx in range(len(node.output)):
+        if idx not in range(len(function_proto.output)):
```

Comment:
Hi, the more important change is: make "input_names_map[function_proto.input[idx]]" the empty string if "idx >= len(node.input)". E.g., iterating over range(len(function_proto.input)) would be fine.